src/features/contactmoment/types.ts

```diff
@@ -9,6 +9,9 @@ export interface ContactmomentViewModel {
   tekst: string;
   zaken: ContactmomentZaak[];
   contactverzoeken: { medewerkers: string[]; completed?: Date }[];
+  "x-commongateway-metadata": {
```

Comment:
can we map this to something more convenient? or would that lead to problems in other areas?